lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -64,6 +65,14 @@ function dosomething_reportback_build_reportbacks_query($params = array()) {
     }
   }
 
+  if (isset($params['runs'])) {
+    if (is_array($params['runs'])) {
+      $query->condition('rb.run_nid', $params['runs'], 'IN');
+    } else {
+      $query->condition('rb.run_nid', $params['runs'], '=');
+    }
+  }
+
   // Public API properties to expose:
```

Comment:
@deadlybutter for this, your PR only has it query for `=` since only allowing to check for one run_nid, which makes sense